python/config.cfg

```diff
@@ -39,7 +39,7 @@ train_conv_mode = fft
 
 # cost function: square_loss, binomial_cross_entropy, softmax_loss, auto
 # auto mode will match the out_type: boundary-softmax_loss, affinity-binomial_cross_entropy
-cost_fn = auto
+cost_fn = square_loss
```

Comment:
The original thought is that it would be easier for new people to interpret/troubleshoot square error instead of having to take an extra step to figure out which error they're even using. Not a huge deal for me either way though. If you greatly prefer auto I'll swap it back